test/modules/fundingManager/bondingCurve/utils/mocks/RedeemingBondingCurveBaseV1Mock.sol

```diff
@@ -24,6 +24,9 @@ import {IERC20} from "@oz/token/ERC20/IERC20.sol";
 contract RedeemingBondingCurveBaseV1Mock is RedeemingBondingCurveBase_v1 {
     IBancorFormula public formula;
 
+    //--------------------------------------------------------------------------
```

Comment:
Missing space between `//` and `---`
(maybe just update the whole file if you are at it?)

Reply:
Done